src/Notification/Notification.php

```diff
@@ -266,4 +274,23 @@ private static function getBlueprintAttributes(BlueprintInterface $blueprint): a
             'data' => ($data = $blueprint->getData()) ? json_encode($data) : null
         ];
     }
+
+    /**
+     * Adds a notification driver to the list.
+     *
+     * @param string $driverName
+     * @param NotificationDriverInterface $driver
+     */
+    public static function addNotificationDriver(string $driverName, NotificationDriverInterface $driver)
```

Comment:
I feel like all these might be better stored on the syncer class. It's needed there, but more importantly, `NotificationSyncer` and the drivers concern methods of delivery / distribution for notifications. `Notification` is just the data structure for persisting them, which is currently agnostic of method of delivery.

Reply:
I actually had them in `NotificationSyncer` at first, but then moved them. Me overthinking stuff :D